sdk/identity/azure-identity/azure/identity/_credentials/azure_powershell.py

```diff
@@ -51,7 +51,14 @@ class AzurePowerShellCredential(object):
     """Authenticates by requesting a token from Azure PowerShell.
 
     This requires previously logging in to Azure via "Connect-AzAccount", and will use the currently logged in identity.
+
+    :keyword List[str] additionally_allowed_tenants: optional additional tenant ids for which the credential
+        may acquire tokens. Add the wildcard value "*" to allow the credential to acquire tokens for
+        any tenant the application is installed.
```

Comment:
For credentials like this, where not setting a default tenant is equivalent to saying "this credential is good for all tenants", I wonder if we should specify that in the docstring

Reply:
We don't want to add the words now because currently we don't allow user to specify tenant_id in the ctor (it is a beta feature which will ship in Oct.)

Reply:
In that case, does `additionally_allowed_tenants` do anything right now? If not, should it be included in the docstring?

Reply:
Given we will GA tenant_id support in Oct, I am afraid we don't want to say additionally_allowed_tenants will not take effect in our docstring now. :)